### PR TITLE
Capstone-2-finish

### PR DESCRIPTION
**Pull request by AnsarIbrahim**
The team was comprised of two individuals. 💯
We have managed to do the following to meet the requirements of the capstone 🗿 project 🎊 :

We determined the external API to implement a page. ✔️
We added a header to the page. ✔️
We added a footer to our page. ✔️
We set up a new app and received an id. ✔️
We fetched data from APIs and used it to establish the homepage and the comments 💬 popup. ✔️
We saved the number of likes per card/meal to an API and retrieved them on page reload. ✔️
We saved comments for each meal recipe on the API and retrieved them on page reload. ✔️
We count the number of items displayed on the homepage and the number of comments pertaining to a recipe. ✔️
We tested the count comments and counted items functions with jest. ✔️
We documented the web app in a video. ✔️
We updated the README file. ✔️

### CODE REVIEW

File: src/test/movie.test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      numberOfSeasons: 3,
+    },
+    {
+      movieName: 'The King',
+      movieId: 3,
+      numberOfSeasons: 7,
+    }];
+
+  expect(database(i)).toBe(i.length);
+});
```
Comment (medaminedev66) [inline src/test/movie.test.js:28]: Please ensure that your project meets the following requirements :
- [ ] Each counter should be implemented as a separate module (Items counter and comments counter).
- [ ] A counter function should look for specific DOM elements (e.g. for the comments counter it should look for comments) and make the counting based on what is actually displayed on the page.
- [ ] These counter functions need to be covered with unit tests using Jest. Make sure that you have tests for all edge cases.
* You should think at least about the scenario in which there are 0 items on the page (counter function should return 0)

File: src/modules/comment.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
+import dbcaller from './databasecaller.js';
+import commentsFunction from './commentFunction.js';
+
+const main = document.querySelector('main');
+
+const aMovie = async (a) => {
```
Comment (medaminedev66) [inline src/modules/comment.js:6]: - [ ] Following JavaScript best practices, a function in JavaScript should have a descriptive name. You should use descriptive nouns and verbs as prefixes. For example, if we declare a function to add a movie, the function name should be addMovie. 
Kindly ensure that all the variables and functions names in your project are descriptive and follow the [naming conventions.](https://www.syncfusion.com/blogs/post/10-javascript-naming-conventions-every-developer-should-know.aspx)

File: README.md [hunk 3]
```diff
@@ -1 +1,207 @@
...
+- **local-storage**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link]()
```
Comment (medaminedev66) [inline README.md:87]: [OPTIONAL] The live demo link is broken. For professional readme, it would be nicer to fix it. If the student decides not to deploy the application, you may delete the live demo section.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
@Ferizad and @AnsarIbrahim 
### This is a manual review
# Request Changes ♻️
Check the comments under the review
# Optional Changes
For a professional repo:
- Pull Request should have a descriptive title telling others what changes are inside. 
- Commit messages should be meaningful.
- Check the comment with [OPTIONAL] prefix under the review

Cheers and Happy coding!:clap::clap::clap:
